features/password_reset.feature

```diff
@@ -46,3 +46,16 @@ Feature: Password Reset
     And I press "Email me a link to reset my password"
     Then I should see "No user was found with that email address"
     And I should be on the password reset page
+
+  Scenario: Trying to reset password to the password currently in the database
+    Given I am on the login page
+    And I follow "Forgot your password?"
+    When I fill in "email" with "affiliate_admin@fixtures.org"
+    And I press "Email me a link to reset my password"
+    Then "affiliate_admin@fixtures.org" should receive an email
```

Comment:
I believe you can omit these lines:
```
    Then "affiliate_admin@fixtures.org" should receive an email
    When I open the email
    And I should see "https://localhost:3000/password_resets" in the email body
```
Those are tested in the preceding scenarios, and should not be necessary to run this feature.